### PR TITLE
Add warning for large files in validate/conversion/convert

### PR DESCRIPTION
**Pull request by rtgdk**
#306

### CODE REVIEW

**Comment by rtgdk** [comment]
Screenshot 2021-06-29 at 4 00 21 PM
@goneall Let me know what to change the file size limit to and warning msg

File: src/app/static/js/warning.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+let MAX_FILE_SIZE = 10485760; // 10 MB in bytes
+let MAX_FILE_SIZE_STR = "10MB";
+function checkFileSize(file, warningElementJQ) {
```
Comment (goneall) [inline src/app/static/js/warning.js:3]: Looking at issue #306 and the attached file, 6MB will cause issues.  Let's go with 5MB as the size for a warning.

**Review by goneall — CHANGES_REQUESTED** [review]
Thanks @rtgdk - just one change request to update the size.  Otherwise, LGTM.

File: src/app/static/js/warning.js (hunk 1 above)
Comment (rtgdk) [reply src/app/static/js/warning.js:3]: Fixed it 👍

**Review by goneall — APPROVED** [review]
Thanks @rtgdk - I'll go ahead and merge

**Comment by goneall** [comment]
@kestewart This is the PR we talked about on the tech call that provides better feedback for large files.

I'll merge the PR, but I'm going to wait to deploy to see if there are other tooling fixes based on the NTIA plugfest.

**Pull request merged**